apps/passport-server/.env.local.example

```diff
@@ -79,3 +79,12 @@ PRETIX_SYNC_DISABLED=
 # If true, you have set up a local cert for dev.local for hot reloading locally with https
 # https://github.com/emergenceland/zupass/blob/test/bot.md#6-optional-hot-reloading-for-tg-development
 IS_LOCAL_HTTPS=false
+
+# Allow a maximum of this many account resets:
+# ACCOUNT_RESET_LIMIT_QUANTITY=5
+# .. Per this amount of time
+# ACCOUNT_RESET_LIMIT_DURATION_MS=86400000
```

Comment:
Can you comment what this works out to in hours, so readers don't have to do the math?

Reply:
done